test/index.js

```diff
@@ -24,6 +24,40 @@ function syncMock () {
 
 
 describe('Semafour', () => {
+  it('can try locking semaphores', (done) => {
```

Comment:
The version of lab was updated. `done` is no longer used. See the surrounding tests. If you have a specific question, just ask.